lib/processes/visTestPattern.cpp

```diff
@@ -124,131 +107,393 @@ void visTestPattern::main_thread() {
 
     uint64_t i_frame = 0;
 
+    // Wait for the first frame
+    if (wait_for_full_frame(in_buf, unique_name.c_str(), frame_id) == nullptr) {
+        if (outfile.is_open())
+            outfile.close();
+        return;
```

Comment:
Do you also want to `raise(SIGINT)` here, or some other error condition?

Reply:
Why? This is a normal exit condition, even though as bit unusual on start. I will add an `INFO`.

Reply:
Oh, sorry, I must have misread that statment.  I agree this is fine as is.